documentation/source/conf.py

```diff
@@ -132,11 +132,15 @@
 }
 
 html_context = {
-    "display_github": True,
-    "github_user": "neuropoly",
-    "github_repo": "spinalcordtoolbox",
-    "github_version": "master",
-    "conf_py_path": "/documentations/",
+    # TODO: when the Github icon is supported natively by furo (https://github.com/pradyunsg/furo/discussions/114)
+    # then this should be moved into html_theme_options and the theme_ prefix should be dropped
+    "theme_source_repository": "https://github.com/spinalcordtoolbox/spinalcordtoolbox",
+    "theme_source_branch": "master",  # or subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]) ?
```

Comment:
Regarding your comment here... say a dev tries to build the docs on a personal branch. What happens?

Hmm... it looks like a warning would be thrown:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d04d45931130d4561e5aa4058f3e45d4e687052e/documentation/source/_templates/partials/edit-this-page.html#L42-L44

I'm surprised that a warning isn't thrown on the branch for this PR, then. Am I misunderstanding something?

Reply:
Oh wait, this only checks if the variable is set, and not if it matches the current branch.

So, `master` is basically a dummy value then, eh?

Reply:
This value just gets pasted together to create the final link: https://github.com/pradyunsg/furo/discussions/114#discussioncomment-1091908

It doesn't have to care about what branch you're actually building it from or if you're even on a branch at all.

Reply:
Ahhh, I see!! Thank you. :)